.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
```

Comment:
Do you think that instead of a single actions, there should be several small purpose ones?

Reply:
You are talking about splitting the one ci.yml file into multiple files, where e.g. one CI job / category of job corresponds to a file, so that it is possible to give more meaningful names? 

I don't know if it has any implications. It is up to preference I guess. This is how I've written / copied github action files so far without giving it much thought, but I can see that a different way is preferred / cleaner. 

For now though, the different steps are easily distinguishable with their current job names and the [CI structure](https://github.com/Sh3Rm4n/sealed-rs/actions/runs/456796219) isn't too complicated.

Reply:
Yes it was precisely that. Since this is a small project this concern seems irrelevant for now